internal/app/api/v1/repository.go

```diff
@@ -58,7 +69,7 @@ func (s TestkubeAPI) ValidateRepositoryHandler() fiber.Handler {
 					}
 
 					if value, ok := data[item.secretRef.Key]; !ok {
-						return s.Error(c, http.StatusBadGateway, fmt.Errorf("missed key %s in secret %s/%s",
+						return s.Error(c, http.StatusBadGateway, fmt.Errorf("username secret or token secret: missed key %s in secret %s/%s",
```

Comment:
can we also add the errPrefix here for consistency?

Reply:
yes, merged with your pr and added prefix